modules/managing-model-registry-permissions.adoc

```diff
@@ -0,0 +1,63 @@
+:_module-type: PROCEDURE
+
+[id="managing-model-registry-permissions{context}"]
+= Managing model registry permissions 
+
+[role='_abstract']
+You can manage access to a model registry for individual users and user groups in your organization, and for service accounts in a project.
+
+{productname-short} creates the `<model-registry-name>-users` group automatically for use with model registries. You can add users to this group in OpenShift, or ask the cluster admin to do so.
+
+.Prerequisites
+* You have logged in to {productname-long}.
+ifdef::upstream[]
+* If you are using {productname-short} groups, you are part of the administrator group (for example, {odh-admin-group}). If you are not using specialized groups, you are part of the {openshift-platform} administrator group.
+endif::[]
+ifndef::upstream[]
+* If you are using {productname-short} groups, you are part of the administrator group (for example, {oai-admin-group}). If you are not using specialized groups, you are part of the {openshift-platform} administrator group. For more information, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/installing-and-deploying-openshift-ai_install#adding-administrative-users-in-{openshift-platform-url}_install[Adding administrative users in {openshift-platform}].
+endif::[]
```

Comment:
Good idea, have updated with your suggestion. Might be a good idea to raise with the team (again?), this would be a good simplification to apply across the docs.

Reply:
I have a jira as part of ENG-12839